addons/editor/functions/fnc_handleSearchClick.sqf

```diff
@@ -26,4 +26,4 @@ private _ctrlSearchEngine = _display displayCtrl IDC_RSCDISPLAYCURATOR_CREATE_SE
 _ctrlSearchEngine ctrlSetText "";
 
 _ctrlSearchCustom ctrlSetText "";
-ctrlSetFocus _ctrlSearchCustom;
+ctrlSetFocus _ctrlSearchCustom
```

Comment:
Old was correct, the function header for this file is incomplete - return value should be "None".